includes/storage-quickstart-tutorial-intro-include-powershell.md

```diff
@@ -17,28 +17,30 @@ Connect-AzAccount
 If you don't know which location you want to use, you can list the available locations. Display the list of locations by using the following code example and find the one you want to use. This example uses **eastus**. Store the location in a variable and use the variable so you can change it in one place.
 
 ```powershell
-Get-AzLocation | select Location
-$location = "eastus"
+Get-AzLocation | Select-Object -Property Location
+$Location = "eastus"
```

Comment:
Single quotes should be used for literals. Double quotes when necessary for anything that needs to be expanded within the quotes such as variables.

Reply:
changed!  I thought I'd caught that!